### PR TITLE
Release version 0.3.2

### PR DESCRIPTION
**Pull request by itchymax**
Hi @da-kami!
This PR was created in response to a manual trigger of the release workflow here: https://github.com/itchysats/10101/actions/runs/3635440547.
I've bumped the versions in the manifest files in this commit: 4569a4e07612da1d06c34c355a3d89740567150d.
Merging this PR will create a GitHub release!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Fixed
+
+- Action button that enables simple navigation to the wallet dashboard from all screens that show the balance.
+- Don't auto open the keyboard in the CFD trading screen.
+- Default invoice description keyboard to text.
+- Persist feed backup state over restarts.
+
+### Added
+
+- Setting screen that shows some useful app information but does not allow changes yet.
```
Comment (luckysori) [inline CHANGELOG.md:21]:
```suggestion
- Settings screen that shows some useful app information but does not allow changes yet.
```

**Review by luckysori — APPROVED** [review]

**Review by klochowicz — APPROVED** [review]
I would love to get #557 in for the release. I think it's a game-changer.

**Pull request merged**